Homework_9_Incapsulation_Inheritance_Polymorphism/My_Company/src/com/example/employees/Employee.java

```diff
@@ -0,0 +1,78 @@
+package com.example.employees;
+
+public class Employee {
+
+    private String name;
+    private int age;
+    private int salary;
+    private static int minSalary = 2000;
+
+    public Employee(String name, int age, int salary) {
+        this.name = name;
+        setAge(age);
```

Comment:
calling setter methods to trigger the specific behavior instead of a direct assignment - good style 👍